scala_proto/scala_proto.bzl

```diff
@@ -32,85 +35,105 @@ def scala_proto_repositories():
       actual = '@scala_proto_rules_protoc_jar//jar')
 
   native.maven_jar(
-      name = "scala_proto_rules_scalapb_plugin",
-      artifact = scala_mvn_artifact(
-          "com.trueaccord.scalapb:compilerplugin:0.6.5"),
-      sha1 = "290094c632c95b36b6f66d7dbfdc15242b9a247f",
-      server = "scala_proto_deps_maven_server",
+      name = "scala_proto_rules_scalapb_plugin_2_11",
+      artifact = "com.trueaccord.scalapb:compilerplugin_2.11:0.6.5",
+      #server = "scala_proto_deps_maven_server",
+  )
+  native.maven_jar(
+      name = "scala_proto_rules_scalapb_plugin_2_12",
+      artifact = "com.trueaccord.scalapb:compilerplugin_2.12:0.6.5",
+      #server = "scala_proto_deps_maven_server",
   )
 
   native.bind(
       name = 'io_bazel_rules_scala/dependency/proto/scalapb_plugin',
-      actual = '@scala_proto_rules_scalapb_plugin//jar')
+      actual = '@scala_proto_rules_scalapb_plugin_{}//jar'.format(major_version_underscore))
 
   native.maven_jar(
-      name = "scala_proto_rules_protoc_bridge",
-      artifact = scala_mvn_artifact(
-          "com.trueaccord.scalapb:protoc-bridge:0.3.0-M1"),
-      sha1 = "73d38f045ea8f09cc1264991d1064add6eac9e00",
-      server = "scala_proto_deps_maven_server",
+      name = "scala_proto_rules_protoc_bridge_2_11",
+      artifact = "com.trueaccord.scalapb:protoc-bridge_2.11:0.3.0-M1",
+      #server = "scala_proto_deps_maven_server",
+  )
+  native.maven_jar(
+      name = "scala_proto_rules_protoc_bridge_2_12",
+      artifact = "com.trueaccord.scalapb:protoc-bridge_2.12:0.3.0-M1",
+      #server = "scala_proto_deps_maven_server",
```

Comment:
let's just remove it entirely vs comment.

Reply:
I noticed that in [twitter_scrooge.bzl](https://github.com/bazelbuild/rules_scala/blob/master/twitter_scrooge/twitter_scrooge.bzl#L21) a maven_server with bazels mirror is specified, do we want to use that one or just remove all `server = ...` from this file (I guess `maven_jar` defaults to maven central)